scripts/kickstart_generator.rb

```diff
@@ -23,17 +23,17 @@ def initialize(build_base, targets, puddle, manageiq_checkout, appliance_checkou
       @ssui_checkout      = ssui_checkout
     end
 
-    def run(task = :all)
+    def run
```

Comment:
I didn't realize that there was a test harness for this code.  I'll have a look at fixing it.